wgsl/index.bs

```diff
@@ -3745,7 +3745,8 @@ Issue: Which index is used when it's out of bounds?
   <tr algorithm="matrix indexed column vector selection">
        <td class="nowrap">
           |e|: mat|N|x|M|&lt;|T|&gt;<br>
-          |i|: [INT]
+          |i|: [INT]<br>
+          |i| is a [literal](#literals)
```

Comment:
maybe add an "Issue(https://github.com/gpuweb/gpuweb/issues/1272): switch the requirement to use constexpr"

Reply:
I mentioned #1272 in the original commit message, but now that I've looked at it more, I'm not sure it's what we really want to refer to here. I assume that, once #1272 is merged, the definition of `const_expr` will be updated appropriately, and we won't need to change a thing here.